packages/epo-react-lib/.storybook/utilities/helpers.ts

```diff
@@ -0,0 +1,21 @@
+export const getGradientImage = (width: number = 200, height: number = 200) => {
```

Comment:
helper that generates a simple gradient image using the Rubin dawn color palette to remove reliance on external placeholder image services